gasprice/gasprice.go

```diff
@@ -87,6 +92,11 @@ func NewGasHelper(config *Config, backend Blockchain) *GasHelper {
 		pricePercentile = 100
 	}
 
+	cache, err := lru.New(100)
+	if err != nil {
+		panic(err)
```

Comment:
There is a `linter` rule that forbids using panics in code. Just return an error from the `NewGasHelper` function, and change the places where we call it to propagate the error.